### DISCUSSION TITLE
Release versions of OpenJDK that are not marked as LTS

### DISCUSSION THREAD

**Discussion by AlbozDroid** [microsoft/openjdk#17] [Q&A] [answered]
Firstly, congratulations on the new Microsoft build of OpenJDK.

We are an Azure customer, and we have a few products built on Java. We'd like to keep our modern Web Apps up to date with the latest improvement of the platform. 
As an example, JDK 14 and 15 were not LTS, but they had big improvements in performance, garbage collection, and memory consumption. Those were enough reasons for us to move to JDK 15.  We now plan to either be on the latest JDK or the one before. 

Will MS release any build in between LTS? Maybe do something like Azul, they seem to make MTS (Medium Term Supported) for some of the releases in between?

Thanks.

**Comment by karianna** [comment, marked as answer]
Thanks for the feedback, at this stage we're only planning on supporting LTS releases but we'd love to hear more from you and others as to what they would like to see beyond that.

Would you for example move from 15 to 16 and then 17 every 6 months or would that not be practical for your use case?

**Reply by AlbozDroid** [reply]
For us, it would be practical to do that. We will move to 16 by the end of this month and then to 17 once it's released.

I understand that old products might not be able to move this quickly, but the gap between LTS releases is too big. 
I wonder whether some middle ground with MTS releases could be found. Just throwing it there as an idea.

Perhaps some alliance with other vendors to release an MTS every year and support it for one year would do the trick (this way we are in the latest or the version before).

**Reply by karianna** [reply]
Thanks for the context!  I'm sending the link of this discussion to other folks that have raised similar questions in the public and see what other common use cases come up.

**Reply by chrismyers81** [reply]
For my part as a Systems guy, I like using vendor-provided and -maintained versions of packages rather than installing and managing my own custom versions. Just makes life easier, so I embrace the LTS versions eagerly. My vendors have a lot more time and resources to vet/patch bugs than I do, so I like just being able to use their update utilities.

**Reply by karianna** [reply]
FYI - The OpenJDK community is converging on 2 year LTS's

**Reply by AlbozDroid** [reply]
Yes, I read Mark Reinold's post. I guess that is a good compromise.
